### PR TITLE
Add `sequentialize_edges`

### PR DESCRIPTION
**Pull request by j042**
Adds `sequentialze_edges()` that works for both lines and polygons. Currently supports searching for separated / individual lines and polygons. With a bit more process, it would be possible to extend this to general polygon search (that'd be for next PR, if required).

Also, made some changes in connec.py to avoid unnecessary copies and sanity checks - This is a ToDo for a follow-up PR to add them as tests.

At last, I deleted previous utils test. I will re-write better suited tests.

@felipegonzalezco

### CODE REVIEW

File: gustaf/utils/connec.py [hunk 1]
```diff
@@ -571,3 +567,253 @@ def sorted_unique(connectivity, sorted_=False):
...
+    polygons = []
+    is_polygon = []
+    for _ in range(len(edges)):
+        polygon = [start_value, other_col[current_id]]
+
+        #        while polygon[0] != polygon[-1]:
+        for _ in range(len(next_candidates)):
+            if polygon[0] == polygon[-1]:
+                break
+            hits = []
```
Comment (clemens-fricke) [inline gustaf/utils/connec.py:690]:
```suggestion
```

Is this line needed?

**Review by clemens-fricke — CHANGES_REQUESTED** [review]
Saw these few things. Would be nice if they could be cleared up. 

Also, we should already add an Issue to add the now removed tests back/add new better tests again.

File: gustaf/utils/connec.py [hunk 2]
```diff
@@ -571,3 +567,253 @@ def sorted_unique(connectivity, sorted_=False):
...
+    line_starts = set(np.where(bc == 1)[0])
+
+    # initialize a set to keep track of processes vertices
+    next_candidates = set(edges.ravel())
+
+    e = collections.namedtuple("a", "b")
+    e.a = edges[:, 0]
+    e.b = edges[:, 1]
+
+    t = collections.namedtuple("a", "b")
```
[the remark this answers is not in the document]
Comment (j042) [reply gustaf/utils/connec.py]: `tree`

File: gustaf/utils/connec.py [hunk 3]
```diff
@@ -571,3 +567,253 @@ def sorted_unique(connectivity, sorted_=False):
...
+            "search. Given edges include connections with more than 3 edges."
+        )
+
+    # we want to keep track of single occurrences, as they are line start
+    line_starts = set(np.where(bc == 1)[0])
+
+    # initialize a set to keep track of processes vertices
+    next_candidates = set(edges.ravel())
+
+    e = collections.namedtuple("a", "b")
```
[the remark this answers is not in the document]
Comment (j042) [reply gustaf/utils/connec.py]: `edge_col`

**Review by clemens-fricke — APPROVED** [review]
Looks good nice job.

**Pull request merged**